### PR TITLE
[PPML] Attestation in PPML E2E: Quote Generation Cmd tool

### PR DESCRIPTION
**Pull request by bronzeMe**
- A sample implementation of the command line tool for quote generation .

### CODE REVIEW

**Comment by qiyuangong** [comment]
Thank you for your contribution!

File: scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/QuoteGeneratorCmd.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import java.io.{File, FileInputStream, FileOutputStream}
+
+object QuoteGeneratorCmd {
+    def main(args: Array[String]): Unit = {
+        val logger = LogManager.getLogger(getClass)
+        val libOSType = System.getProperty("LIBOS_TYPE", "gramine")
+        val userReportDataPath = System.getProperty("REPORT_DATA", "./userReportData")
+        val quoteOutputPath = System.getProperty("QUOTE_PATH", "./quoteOutputDump")
+
+        if(libOSType=="gramine") {
```
[the remark this answers is not in the document]
Comment (bronzeMe) [reply scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/QuoteGeneratorCmd.scala:12]: Thanks for this comments, I will update it soon.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Comment by qiyuangong** [comment]
Thank you for your contribution, @bronzeMe ! :)

**Pull request merged**